### PR TITLE
Doc: clearer basics

### PR DESCRIPTION
**Pull request by pbougue**
"Approach" and "Who's who" sections should be duplicated in README ?

TODO:
- [x] Add to README
- [x] Add that some fields of GTFS/NTFS are ignored and can thus be found on opendatasoft > Not immediately
- [x] Change "endpoint" to "feature" to stick with playground? (mention object/feature difference?)
- [x] Mention id/uri in value > Not immediately

### CODE REVIEW

File: documentation/slate/source/includes/welcome.md [hunk 1]
```diff
@@ -4,18 +4,34 @@ Getting started
...
 * and sexy things such as isochrones
 
-navitia is a [HATEOAS](http://en.wikipedia.org/wiki/HATEOAS) API that returns JSON formated results, available at <https://api.navitia.io>
+Have a look at the examples below to learn what services we provide and how to use them.
+
+
+### Approach
+
+Navitia is an open-source web API, **initially** built to provide traveler information on urban transportation networks.  
+It's main purpose is to provide day-to-day informations to travelers.
```
Comment (stifoon) [inline documentation/slate/source/includes/welcome.md]: "**Its**" main purpose

File: documentation/slate/source/includes/welcome.md [hunk 2]
```diff
@@ -4,18 +4,34 @@ Getting started
...
 
-navitia is a [HATEOAS](http://en.wikipedia.org/wiki/HATEOAS) API that returns JSON formated results, available at <https://api.navitia.io>
+Have a look at the examples below to learn what services we provide and how to use them.
+
+
+### Approach
+
+Navitia is an open-source web API, **initially** built to provide traveler information on urban transportation networks.  
+It's main purpose is to provide day-to-day informations to travelers.  
+Over time Navitia has become doing way more, _sometimes_ for technical and debuging purpose _or_ because other functional needs fit quite well in what Navitia can do _or_ just because it was easy and super cool.
```
Comment (stifoon) [inline documentation/slate/source/includes/welcome.md]: Over "**the**" time Navitia has become
But this sentence is weird, I can't understand it. Maybe you should use google translate to rewrite it? :)

**Review by stifoon — CHANGES_REQUESTED** [review]

**Comment by pbougue** [comment]
I trust more @nonifier remarks on English...
Maybe @TeXitoi and @stifoon have an opinion if I should duplicate it on the README ?
Anyway, expect some delays for revisions on this PR, has the work here is disturbed by a big (BIG) amount of fresh :snowflake: all over the :mountain_snow: , both on slopes and off-pist, so :skier: and :tada: is compulsory :-/

File: documentation/slate/source/includes/examples.md [hunk 3]
```diff
@@ -8,6 +8,34 @@ You will have to use your own token with the examples below.
...
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/stop_areas/stop_area%3ARAT%3ASA%3ABASTI/lines/line%3ARAT%3AM5/departures?count=4&depth=2&' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+```
+
+A query to Navitia's API is divided in 4 parts, as highlighted by colors in a [Navitia Playground example](http://canaltp.github.io/navitia-playground/play.html?request=https%3A%2F%2Fapi.navitia.io%2Fv1%2Fcoverage%2Fsandbox%2Fstop_areas%2Fstop_area%253ARAT%253ASA%253ABASTI%2Flines%2Fline%253ARAT%253AM5%2Fdepartures%3Fcount%3D4%26depth%3D2%26&token=3b036afe-0110-4202-b9ed-99718476c2e0):  
+![Navitia basic request](/images/navitia_basic.png)
+
+1. **Root url** of the API, the adress of the server.  
+Here `https://api.navitia.io/v1/`
+2. **Path**, used to filter the request and precise what is affected by the query. This filter is an instersection of multiple `key/value` (logical _AND_).  
+Here `/coverage/sandbox/stop_areas/stop_area:RAT:SA:BASTI/lines/line:RAT:M5/` means we are looking for informations on everything that is in the region _"sandbox"_ and that is stricly related to both station _"Bastille"_ and line _"metro 5"_.
```
Comment (nonifier) [inline documentation/slate/source/includes/examples.md]: `informations` always is singular ;)

File: documentation/slate/source/includes/examples.md [hunk 4]
```diff
@@ -8,6 +8,34 @@ You will have to use your own token with the examples below.
...
+3. **Endpoint**, specifies what type of information is requested, so the _service_. It can be _journeys_, _isochrones_, _places_...  
+Here `/departures?` means we are requesting _"next departures"_.
+4. **Parameters**, used to precise any additional detail linked to the endpoint requested.
```
Comment (nonifier) [inline documentation/slate/source/includes/examples.md]: `precise` is an adjective (even though it's a verb in French :p ). You could probably use `refine` here.

**Review by stifoon — APPROVED** [review]

**Review by mbounabi — APPROVED** [review]

File: documentation/slate/source/includes/examples.md [hunk 5]
```diff
@@ -8,6 +8,34 @@ You will have to use your own token with the examples below.
...
+# Web is too shiny: JSON, urlencode and curl forever!
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/stop_areas/stop_area%3ARAT%3ASA%3ABASTI/lines/line%3ARAT%3AM5/departures?count=4&depth=2&' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+```
+
+A query to Navitia's API is divided in 4 parts, as highlighted by colors in a [Navitia Playground example](http://canaltp.github.io/navitia-playground/play.html?request=https%3A%2F%2Fapi.navitia.io%2Fv1%2Fcoverage%2Fsandbox%2Fstop_areas%2Fstop_area%253ARAT%253ASA%253ABASTI%2Flines%2Fline%253ARAT%253AM5%2Fdepartures%3Fcount%3D4%26depth%3D2%26&token=3b036afe-0110-4202-b9ed-99718476c2e0):  
+![Navitia basic request](/images/navitia_basic.png)
+
+1. **Root url** of the API, the adress of the server.  
+Here `https://api.navitia.io/v1/`
+2. **Path**, used to filter the request and precise what is affected by the query. This filter is an instersection of multiple `key/value` (logical _AND_).
```
Comment (TeXitoi) [inline documentation/slate/source/includes/examples.md]: instersection?

Comment (TeXitoi) [reply documentation/slate/source/includes/examples.md]: coverage is a bit special

**Review by TeXitoi — CHANGES_REQUESTED** [review]

**Review by nonifier — APPROVED** [review]

**Review by TeXitoi — APPROVED** [review]

File: readme.rst [hunk 6]
```diff
@@ -19,20 +19,42 @@ Welcome to the Navitia repository. Navitia is a service providing:
...
 
-For more information see
+Approach
+----------
+
+Navitia is an open-source web API, **initially** built to provide traveler information on urban transportation networks.  
+Its main purpose is to provide day-to-day informations to travelers.  
+Over time, Navitia has been able to do way more, _sometimes_ for technical and debuging purpose _or_ because other functional needs fit quite well in what Navitia can do _or_ just because it was quite easy and super cool.
+
+Technically, Navitia is a [HATEOAS](http://en.wikipedia.org/wiki/HATEOAS) API that returns JSON formated results.
```
Comment (TeXitoi) [inline readme.rst:36]: md link

File: readme.rst [hunk 7]
```diff
@@ -19,20 +19,42 @@ Welcome to the Navitia repository. Navitia is a service providing:
...
+
+
+Who's who
+----------
+
+
+Navitia is instanciated and exposed publicly through [api.navitia.io](http://api.navitia.io).
```
Comment (TeXitoi) [inline readme.rst:43]: md link

**Pull request merged**